index.html

```diff
@@ -1,13 +1,96 @@
 <!DOCTYPE html>
 <html>
-  <head>
+  <head class="header">
```

Comment:
Since the `<head>` element isn't actually visible on the page, I don't think you need a class name here. The head of the document is just for meta information and links.